content/en/docs/tasks/debug/debug-cluster/troubleshoot-kubectl.md

```diff
@@ -0,0 +1,144 @@
+---
+reviewers:
+# Should be added by an experienced contributor
+title: "Troubleshooting kubectl"
+content_type: task
+weight: 10
+---
+
+<!-- overview -->
+
+This documentation is about investigating and diagnosing
+{{<glossary_tooltip text="kubectl" term_id="kubectl">}} related issues.
+If you are unable to access `kubectl` or unable to connect to your cluster using
+`kubectl`, this document describes several common scenarios and potential solutions
+to identify and address the likely cause.
```

Comment:
Please consider refactor the above sentence ...
"If you are unable to do something, (then) this document describes ...",
the logic here reads confusing to me.